sema/subyt/sinks.py

```
@@ -106,7 +106,7 @@ def add(
         source_mtime: float | None = None,
     ):
         assert self._fopen is not None, "File to Sink to already closed"
-        log.info(f"Creating {self._file_path}")
+        log.info(f"SingleFileSink adding part to {self._file_path}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)